lint/action.yaml

```
@@ -0,0 +1,32 @@
+name: Python Lint
+description: GitHub Action that lints a Python based repository
+inputs:
+  checkout-repo:
+    required: false
+    description: Perform checkout as first step of action
+    default: "true"
+  pypi-token:
+    required: false
+    description: PyPI token that can checkout the repository as well as create tags/releases against it. e.g. 'secrets.PYPI_TOKEN'
+  github-token:
+    required: true
+    description: GitHub token that can checkout the repository. e.g. 'secrets.GITHUB_TOKEN'
+    default: ${{ github.token }}
+runs:
+  using: composite
+  steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      if: inputs.checkout-repo
```

We've seen inconsistent behavior with boolean-ish checks in `if:`. So we make it explicit.

```suggestion
      if: inputs.checkout-repo == 'true'
```